### PR TITLE
Add solution for 853. Car Fleet in Kotlin

### PR DESCRIPTION
**Pull request by t3chkid**
[//]: # "Pull Request Template"
[//]: # "Replace the placeholder values in the template below"

- **File(s) Modified**: _853-Car-Fleet.kt_
- **Language(s) Used**: _Kotlin_
- **Submission URL**: _https://leetcode.com/submissions/detail/811044421/_

[//]: # "Getting the Submission URL"
[//]: # "Go to the leetcode [`Submissions tab`](https://user-images.githubusercontent.com/71089234/180188604-b1ecaf90-bf27-4fd6-a559-5567aebf8930.png)"
[//]: # "and [click on the `Accepted` status of your submission.](https://user-images.githubusercontent.com/71089234/180189321-1a48c33f-aa65-4b29-8aaa-685f4f5f8c9e.png)]"
[//]: # "Finally copy the URL from the nav bar, it should look like https://leetcode.com/submissions/detail/xxxxxxxxx/"

### CODE REVIEW

File: kotlin/853-Car-Fleet.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package kotlin
```
Comment (diss1993) [inline kotlin/853-Car-Fleet.kt:1]: You can remove this package - don't need it in Leetcode

File: kotlin/853-Car-Fleet.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                timeRequiredForCarInFrontToReachInTarget = timeRequiredForCurrentCarToReachTarget
+            }
+        }
+        return numberOfFleets
+    }
+}
+
+
+
+}
```
Comment (diss1993) [inline kotlin/853-Car-Fleet.kt:27]: Remove Extra bracket

**Review by diss1993 — CHANGES_REQUESTED** [review]

File: kotlin/853-Car-Fleet.kt (hunk 1 above)
Comment (t3chkid) [reply kotlin/853-Car-Fleet.kt]: Sure

File: kotlin/853-Car-Fleet.kt (hunk 2 above)
Comment (t3chkid) [reply kotlin/853-Car-Fleet.kt:21]: > 

Sorry about that. Thanks for pointing that out.

**Pull request merged**